StylePropertyMap: Implemented setting sequences of StyleValues in the set method

njlawton opened the pull request.

Commit:
Fixed incorrect CSS property enteries in the PropertyDictionary class
Some properties stored in the dictionary were either stored in a fashion that was inconsistant with how they are called in javascript and others were unable to be set using java script and hence should not have been in the map
thease properties were replaced with ones that had the same characteristics so that the tests relating to them were relevant and passed

Commit:
Implemented setting sequences of StyleValues using the set method
Also implemented basic test

Commit:
Merge branch 'master' into Style-map-set-sequence

Commit:
Merge branch 'Style-map-append' into Style-map-set-sequence

Commit:
Revert "Fixed incorrect CSS property enteries in the PropertyDictionary class"

This reverts commit bb19c12b34d27e98de0a14c72158b71ab8b745c8.

Commit:
Reverted a change in how css properties are stored in the PropertyDictionary

Commit:
Merge branch 'master' into Style-map-set-sequence

wilddamon commented on test/js/inline-style-property-map.js:
```diff
@@ -21,6 +21,16 @@ suite('Inline StylePropertyMap', function() {
+    this.element.style['animation-iteration-count'] = 'infinite, 2, 5';
+    inlineStyleMap.set('animation-iteration-count', valueArray);
+
+    assert.strictEqual(this.element.style['animation-iteration-count'], '4, 5, infinite');
```
Should also test that attempting to set a list of values doesn't work for properties that don't support sequences.

wilddamon commented on test/js/inline-style-property-map.js:
```diff
@@ -21,6 +21,16 @@ suite('Inline StylePropertyMap', function() {
   });
 
+  test('The set method successfully sets the CSS string of the list of StyleValues on property ' +
+    'if it supports setting lists of values', function() {
```
For brevity:
"Set successfully sets the style string for the list of StyleValues on a property that supports sequences."

Commit:
Added test case for the set method
This ensures a type error is thrown when a property that does not support sequences of values is set a sequence of style values

wilddamon commented:
LGTM
